### ISSUE TITLE
Sticky Header/Footer in v1.0.6.4

### ISSUE THREAD

**Issue by NeilRenwick** [mehrgithub/dcrmeg#36]
Hi Mehrdad,

I've just upgraded from v1.0.6.0 to 1.0.6.4 and the sticky header and footer doesn't seem to be present in this version. Has this been removed or is it a configurable option now? I thought it was a great addition so would be a shame if it's gone.

Thanks

**Comment by mehrgithub** [comment]
Hi Neil,

No, I haven't removed the sticky header/footer functionality. I just tested it on 2016 and 365 on premise using IE 11, Chrome, FF, and Opera without any issues.

A few questions:

Do you see the same behavior in different browsers?
Do you see any errors related to the grid (dcrmeg...) in the browser's developer tools console?
What version of CRM are you using?

Thank you,
Mehrdad

**Comment by NeilRenwick** [comment]
Hi Mehrdad,

We're on CRM 2015 which I usually use via IE 11 but have also just tested in Firefox and the header and footer do not seem to be sticky in either browser. I've checked the developer tools console and there are no signs of any errors related to the grid. Please let me know if you want me to check anything else.

Thanks

**Comment by mehrgithub** [comment]
Hi Neil,

Thank you. I will run some more tests specifically on 2015 and will keep you posted.

Mehrdad

**Comment by NeilRenwick** [comment]
Thanks Mehrdad, much appreciated

**Comment by mehrgithub** [comment]
Hi Neil,

I did run many tests on CRM 2015 (DB 7.0.2.53) using new and old organizations. Tested browsers IE 11, Chrome, and FF. I was not able to re produce the issue you have reported.

Please download and import the new release v1.0.6.5 and let me know if this issue still exists.

https://github.com/mehrgithub/dcrmeg/releases/tag/v1.0.6.5

Thank you,
Mehrdad

**Comment by NeilRenwick** [comment]
Thanks Mehrdad. I've been doing some further testing this morning and figured out the cause of the problem which seems to be when using the wrap text option for the column header. If this results in the header running over more than two lines then this results in a scroll bar being added to the grid cutting off part of the header or footer and making it appear that the sticky header/footer isn't working correctly. It doesn't seem to matter how may rows I set for the grid in the Iframe, the issue persists, however this doesn't happen in v 1.0.6.0. I have upgraded to v1.0.6.5 and still have the problem in but I've just re-adjusted my column widths to ensure all headers fit within two lines in order to get around the issue.
